tvm/src/schedule/schedule_dataflow_rewrite.cc

```diff
@@ -579,201 +500,49 @@ Array<Tensor> Schedule::move_to(const Tensor& target, Stage parent,
     }
   }
 
-  // create sender that writes into streaming channel
-  Array<Tensor> consumer_inputs;
-  Array<Buffer> consumer_input_placeholders;
-  Array<Buffer> consumer_output_placeholders;
-  std::string consumer_name = target->op->name + ".channel";
-  if (parent.defined()) consumer_name = target->op->name + ".update.channel";
-
-  Buffer channel_buffer = BufferNode::make(
-      Var(consumer_name, Handle()), target->dtype, target->shape, Array<Expr>(),
-      Expr(), consumer_name, "", 0, 0);
-
-  // move placeholder to or from device
-  if (!parent.defined()) {
-    consumer_inputs.push_back(target);
-    consumer_input_placeholders.push_back(target_buffer);
-    // move data modifed in parent stage
+  // Save the attribute information
+  Interface endpoint(storage, stream_type, mem_port, channel_depth, burst_len,
+                     target->op->name);
```

Comment:
What is an endpoint? A leaf in CDFG?

Reply:
It refers to the nodes that are on the boundary of the subgraph.